internal/indexer/indexer.go

```diff
@@ -854,6 +860,95 @@ func (i *Indexer) indexReferenceToExternalDefinition(p *packages.Package, docume
 	return rangeID, true
 }
 
+// indexImplementations emits data for each implementation of an interface.
+func (i *Indexer) indexImplementations() error {
+	config := &packages.Config{
+		Mode: loadMode,
+		Dir:  i.projectRoot,
+		Logf: i.packagesLoadLogger,
+	}
+
+	deps := []*packages.Package{}
+	for _, dep := range i.dependencies {
+		fmt.Println("-----------", dep.Name)
+		name := dep.Name
+		name = strings.TrimPrefix(name, "https://")
+		name = strings.TrimPrefix(name, "https:/")
+		pkgs, err := packages.Load(config, name)
```

Comment:
lol @tjdevries you were right

Reply:
of course, but about what? :laughing: We saw this while we were testing stuff out today and figured we had to do something like this.

Reply:
I gave you crap about us only calling `package.Load` once and future proofing with that cache 😃